.github/workflows/ping-other-repos.yml

```
@@ -11,6 +11,7 @@ permissions:
 jobs:
   ping:
     runs-on: ubuntu-latest
+    if: ${{ github.actor == 'dependabot[bot]' }}
```

I think this should be reversed:

```suggestion
    if: ${{ github.actor != 'dependabot[bot]' }}
```

---

oofs yeah, thanks for catching that